TheForceEngine/TFE_FrontEndUI/frontEndUi.cpp

```diff
@@ -2231,6 +2234,37 @@ namespace TFE_FrontEndUI
 
 		TFE_Audio::setVolume(sound->soundFxVolume);
 		TFE_MidiPlayer::setVolume(sound->musicVolume);
+
+		int MIDI_DeviceCount = TFE_MidiDevice::getDeviceCount();
+		const char* MIDI_Devices;
+		std::string MIDI_DevicesStr;
+		
+		for (int i = 0; i < MIDI_DeviceCount; i++) {
+			MIDI_DevicesStr += TFE_MidiDevice::getDeviceNameStr(i) + '\0';
+		}
+		MIDI_Devices = &MIDI_DevicesStr[0];
+		
+		ImGui::Dummy(ImVec2(0.0f, 20.0f));
+
```

Comment:
Maybe add an ImGui::Separator() here.

Reply:
I made a commit that I think makes this look OK without the separator.  I'm trying to keep the overall appearance of the sound config menu consistent.